mobilitydb_sqlalchemy/types/Period.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+
+from pymeos.temporal import TText
+from sqlalchemy.types import UserDefinedType
+
+from .TBaseType import TBaseType
+
+def period_ident(self,t):
+    return(t)
+
+class Period(TBaseType):
```

Comment:
I think we can directly inherit from `UserDefinedType` here, as we don't really use anything from `TBaseType`'s functionality here.
```suggestion
class Period(UserDefinedType):
```
We would then not need `pymeos_instant_type` and `pymeos_deserializer_type`, and nor would we need to import `TText`.